frontend-zs/src/components/projects/Home/Map.js

```diff
@@ -0,0 +1,60 @@
+import React, { Component } from 'react';
+
+class Map extends Component {
+    constructor(props) {
+        super(props);
+        this.onScriptLoad = this.onScriptLoad.bind(this)
+    }
+
+    onScriptLoad() {
+        const map = new window.google.maps.Map(document.getElementById(this.props.id), this.props.options);
+        this.props.projects.forEach(project => {
+            new window.google.maps.Marker({
+                position: { lat: project.location[0], lng: project.location[1] },
+                map: map,
+                title: project.title
+            })
+        });
+    }
+
+    componentDidMount() {
+        if (!window.google) {
+            this.callScript()
+        }
+        else {
+            this.onScriptLoad()
+        }
+    }
+
+    callScript = () => {
```

Comment:
```suggestion
    setupMapScript = () => {
```

We need a something more descriptive than `callScript`. `setupMapScript` sounds better for me.